modules/FIC_FileHandler.py

```diff
@@ -30,11 +33,21 @@ def save(self, directory, file_name, content):
         if file_name.endswith(".md"):
             pass
         elif file_name.endswith(".json"):
-            import json
             with open(path, "w") as json_file:
                 json.dump(content, json_file, indent=2)
 
         else:
             print("File Extension is not supported.")
             exit(2)
         pass
+
+    def load_repos(self):
+        from modules.config import REPOSITORIES_FILE
+        import json
+        self.repos_container = json.loads(open(REPOSITORIES_FILE).read())
+
+    def extract_local_data(self):
+        with open(self.data_folder + self.repo_name + ".json", "r") as file:
```

Comment:
You have overwritten the python standard function `file`. 
Based on Official Python documentation, you could say `as f`